### ISSUE TITLE
Add Twitter feed

### ISSUE THREAD

**Issue by asztell** [BitwiseGuy/travel-aegis#2]
- [ ] rewrite Twitter feed template from backend to work with handlebars
- [ ] modify frontend of Twitter feed to work with html/hbs
- [ ] create template component with adminLTE format

**Comment by asztell** [comment]
We should use both MongoDB and MySQL to make it easy to use the risk database.

**Comment by BitwiseGuy** [comment]
Yeah that’ fine. Whatever works best with the data. I think all of the risk data is formatted for SQL anyway.
